example/multiparty-example/multiparty-linkage-in-entity-service.ipynb

```diff
@@ -0,0 +1,1429 @@
+      "\n",
+      "0 ['zachary', 'rawlings', '02-11-1976', 'male', 'melbourne', '180329.075', '08 1694 9465']\n",
+      "2 ['zachary', 'zilm', '02-11-1925', 'male', 'melbourne', '187676.608', '07 5052 9366']\n",
+      "\n",
+      "2 ['georgia', 'mccarthy', '16-03-1916', 'male', 'melbourne', '196008.922', '02 0911 5640']\n",
+      "4 ['georgia', 'mccarthy', '11-07-1969', 'female', 'melbourne', '554992.383', '04 3663 5077']\n",
+      "\n",
+      "0 ['molly', 'locge', '', 'male', 'emlbovrne', '168663.495', '']\n",
+      "3 ['mvolly', 'lodge', '', 'malr', 'melbourne', '168663.495', '']\n",
+      "\n",
+      "1 ['abbey', 'lansdell', '28-08-1994', 'male', 'canberra', '71457.982', '08 5152 2431']\n",
+      "2 ['abby', 'lansdell', '28-09-2994', 'male', 'canberra', '71457.98q', '']\n",
+      "\n",
+      "0 ['sam', 'campbell', '22-07-1989', 'male', 'melbourne', '137253.404', '07 5381 5883']\n",
+      "3 ['ruby', 'campbell', '29-04-1967', 'male', 'melbourne', '170285.986', '07 3781 4180']\n",
+      "\n",
+      "0 ['alexander', 'clarke', '01-10-2011', 'male', 'perth', '428413.506', '02 1306 2910']\n",
+      "2 ['aelxander', 'clarke', '', 'male', '', '428413.506', '02 1306 2010']\n",
+      "\n",
+      "0 ['ella', 'pottjcary', '', 'female', 'brshane', '47026.887', '08 1137 9465']\n",
+      "4 ['ella', 'potticar', '', 'femaoe', '', '47026.887', '08 1137 9455']\n",
+      "\n",
+      "2 ['kelsey', 'campjbelkr', '', 'malr', 'canbrrra', '70454.780', '04 1170 0228']\n",
+      "4 ['kelsey', 'campbell', '18-03-2000', 'male', 'canberra', '70454.780', '04 1170 0228']\n",
+      "\n",
+      "2 ['alec', 'dibben', '15-12-1943', 'femqle', '', '222948.918', '08 3006 8457']\n",
+      "3 ['alec', 'dibben', '15-12-1942', 'femsle', 'mell>oume', '222949.9q8', '']\n",
+      "\n",
+      "0 ['rileh', 'mercorella', '11-10-2970', 'male', 'melbour17e', '40957.245', '08 3535 3768']\n",
+      "3 ['riley', 'merco5ekla', '11-10-1970', 'male', '', '40957.345', '08 2525 2768']\n",
+      "\n",
+      "4 ['xepheern', 'manfonk', '27-10-1944', 'mzo2', '', '65485.672', '04 5681 6529']\n",
+      "2 ['xeperen', 'majfonk', '27-10-1944', 'male', '', '65485.672', '']\n",
+      "\n",
+      "1 ['william', 'held', '15-10-1930', 'female', 'brisbane', '55792.402', '02 8260 1379']\n",
+      "3 ['william', 'spargo', '26-10-1930', 'female', 'brisbane', '28700.849', '03 2206 3864']\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "for group in itertools.islice(groups[\"groups\"], 20):\n",
+    "    for i, j in group:\n",
+    "        print(i, datasets[i][j])\n",
+    "    print()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This matching appears impressively accurate for such noisy data. However, there is still clearly some room for improvement. We may be able to improve on this results by fine-tuning the hashing schema or by changing the threshold."
```

Comment:
The user doesn't know how noisy this data is, so (s)he might not be that impressed. Also, `impressively accurate` is astonishingly vague... You could quantify the accuracy.
For the improvement part, why don't you report the results for other thresholds as well?
Maybe point to an artifact where we do some sort of fine-tuning of the hashing schema.